src/pyggdrasil/scripts/cell_simulation.py

```diff
@@ -0,0 +1,398 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Generate cell mutation matrices,
+by generating random trees and sampling cell attachments
+with noise if error rates are non-zero.
+
+As per definition in the SCITE Jahn et al. 2016.
+"""
+
+import argparse
+import jax.random as random
+import networkx as nx
+import numpy as np
+from jax.random import PRNGKeyArray
+import json
+import os
+
+from pyggdrasil.tree import TreeNode
+from pyggdrasil.serialize._to_json import serialize_tree_to_dict
+import pyggdrasil.tree_inference._simulate as sim
+
+
+def create_parser() -> dict:
```

Comment:
Yes, great work! I haven't actually thought about making it interactive (which I have done only once so far).

A stylistic thing for the future: instead of converting `argparse.Namespace` to `dict`, you can just return it and use `args.seed` rather than `dict_of_args["seed"]`.

(Educational comment: feel free to resolve without taking any action.)